packages/react/src/components/LocaleModal/LocaleModalCountries.js

```diff
@@ -28,6 +28,11 @@ const LocaleModalCountries = ({
   setClearResults,
   ...modalLabels
 }) => {
+  const localList = useRef(null);
+  useEffect(() => {
+    localList.current.scrollTop = 0;
+  });
```

Comment:
`useEffect()` with no "dependency arg" causes running the callback upon every render. Looks that we need to define `open` prop in `<LocaleModalCountries>` and have `<LocaleModal>` pass along the open state:

```suggestion
  useEffect(() => {
    if (open) {
      localList.current.scrollTop = 0;
    }
  }, [open]);
```